otel-extension/src/it/scala/io/scalac/mesmer/otelextension/instrumentations/zio/ZIOMetricsTest.scala

```diff
@@ -109,4 +148,9 @@ class ZIOMetricsTest
   private def getCounterAttributes(data: MetricData): Option[Attributes] =
     data.getDoubleSumData.getPoints.asScala.map(_.getAttributes).toList.headOption
 
+  private def assetHistogramData(metricName: String, block: HistogramData => Unit): Unit =
```

Comment:
Not necessarily for now, but in general: the "assert..." methods could be extracted to some helper class